test/source/parentClickViewGenerator.js

```diff
@@ -16,8 +16,8 @@ module.exports = function(window) {
     events: {
       'click div.parent' : 'myClick'
     },
-    render: function() {
-      this.$el.html("<div class='parent'>test</div><div inject='one'></div><div inject='two'></div>");
+    template: "<div class='parent'>test</div><div inject='one'></div><div inject='two'></div>",
+    attachTrackedViews: function() {
```

Comment:
The dissonance between having attachTrackedViews use injectView instead of attachView is confusing - will create and additional issue regarding naming